.travis.yml

```diff
@@ -12,7 +12,7 @@ script:
 jobs:
   include:
   - stage: Lint and Test
-    script: npm run lint && npm run docs:lint && npm run test:ci-config && npm run test
```

Comment:
Removing this for now since the linting is more strict. I'll create a follow-up to lint the `docs/` folder too, but didn't want to increase the scope.